lib/connectors/database.js

```diff
@@ -17,6 +17,28 @@ database.prototype._queryWithValues = function(opts, callback){
   return pool.query(opts.sql, opts.values, callback);
 };
 
+database.prototype.checkForParagraphs = function(doc, callback) {
+  if(!doc) return callback(new VError('doc parameter not passed to checkForParagraphs.'));
+  if(doc === '') return callback(new VError('Empty doc parameter not passed to checkForParagraphs.'));
+
+  var docsWithSubdirectories = '%' + doc + '%';
```

Comment:
This works fine as long as there are no documents in the parent level with the same name. We will ensure to make this point during training. There aren't any as it is anyway.